sct-commons/src/test/java/org/lfenergy/compas/sct/commons/ExtRefEditorServiceTest.java

```diff
@@ -20,9 +21,11 @@
 import org.lfenergy.compas.sct.commons.scl.SclRootAdapter;
 import org.lfenergy.compas.sct.commons.scl.ldevice.LDeviceAdapter;
 import org.lfenergy.compas.sct.commons.scl.ln.AbstractLNAdapter;
+import org.lfenergy.compas.sct.commons.testhelpers.MarshallerWrapper;
```

Comment:
unused import

Reply:
Thanks, I did not configure the "commit Optimize import' on this project.